hook!-full-game/src/Player/Hook/Skills/Base.gd

```diff
@@ -25,16 +24,33 @@ func _unhandled_input(event: InputEvent) -> void:
 
 
 func _hook() -> void:
+	if not hook.cooldown.is_stopped():
+		return
+	_snap_hook_arrow_tip()
+	_disable_hook_aim()
```

Comment:
We should use properties instead: `hook.aim_mode = false`

Reply:
Same thing here, we'll merge as is and I'll go over later when switching to HFSM